jormungandr/src/rest/v0/logic.rs

```diff
@@ -658,3 +659,12 @@ pub async fn get_committees(context: &Context) -> Result<Vec<String>, Error> {
         .map(|cid| cid.to_string())
         .collect())
 }
+
+pub async fn get_active_vote_plans(context: &Context) -> Result<Vec<VotePlan>, Error> {
+    Ok(context
+        .blockchain_tip()?
+        .get_ref()
+        .await
+        .ledger()
+        .active_vote_plans())
```

Comment:
there is a function `active_vote_plans` from the `Ref`, why using the indirection from the `ledger()`?

Reply:
Leftover from a prev version. Thanks for pointing out.